### DISCUSSION TITLE
Stuck on move a folder inside option

### DISCUSSION THREAD

**Discussion by howisirktaken** [The-Wineskin-Project/WineskinServer#245] [Q&A] [answered]
Hello. Currently trying to run a private server WoW with Wineskin. For some reason the  "move a folder" option not might not be working. It would just lead me into an infinite loop of "wineskin is currently busy please wait". I waited for about 2-3 hours and nothing happened. Also, when I would try to use the browse button to look for an .exe file it also won't work.

**Comment by Gcenx** [comment]
This is related to #244 

I won’t get time to look into this until the weekend at the earliest, didn’t really have much option but to push this wrapper before I was done testing

**Comment by Gcenx** [comment, marked as answer]
Actually done a quick look over and believe I’ve resolved the issues so pushed an update.

**Reply by howisirktaken** [reply]
Will try it again when I get back home. Thank you so much.

**Reply by howisirktaken** [reply]
I actually just tried it and it is now properly working. You are the best. Again, thank you.
